server/public/stylesheets/style.css

```diff
@@ -6,3 +6,18 @@ body {
 a {
   color: #00B7FF;
 }
+nav#serverNav{
+  margin-bottom: 20px;
+  padding-bottom: .53rem;
+  
+ }
+ button,.navbar-toggler{
+   margin-top:-5px;
+  }
+  .addNewArticle{
+    margin-top: 20px;
+    margin-bottom: 20px;
+  }
+  .visHid{
```

Comment:
don't use names like that `visHid` - you need to think about naming you css classes the same way you think about javascript variables.